mergify_engine/gh_pr_fullifier.py

```diff
@@ -114,11 +108,15 @@ def disabled_by_rules(pull, **extra):
 
 def compute_weight_and_status(pull, **extra):
     status_desc = disabled_by_rules(pull, **extra)
+    reviews_ok, reviews_ko, reviews_required = pull.mergify_engine["approvals"]
     if status_desc is not None:
         weight = -1
```

Comment:
Maybe we should add, to be clear:

```
if reviews_ko:
    weight = -1
    status_desc = "%d changes requested" % len(reviews_ko)
```

Reply:
I chose to change here to ignore reviews_ko, as IIUC GitHub does.
Not ignoring them could be another option in the yaml file.

Reply:
I missed you other reply. I did not understood GitHub was counting reviews KO. Does it count it the same way as you did or just one KO blocks the entire PR?

Reply:
Like I did, one KO and that blocks the PR